### ISSUE TITLE
Fails to load v 0.10.16 and 0.10.17

### ISSUE THREAD

**Issue by cscott** [cortexrd/Knack-Toolkit-Library#12]
with both 'min' and 'full' I get the following in the Firefox console:
```
InstallTrigger is deprecated and will be removed in the future. [KTL-0.10.17.js:8752:29](https://ctrnd.com/Lib/KTL/Prod/KTL-0.10.17.js)
Uncaught ReferenceError: inlineEditActive is not defined
    KnackApp https://ctrnd.com/Lib/KTL/KTL_KnackApp.js:104
    KnackApp https://ctrnd.com/Lib/KTL/KTL_KnackApp.js:186
    loadKtl https://ctrnd.com/Lib/KTL/KTL_Start.js:67
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    js https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    KnackInitAsync https://appcdn.cloud-database.co/642ee87912e0ca00294f3798/custom/main.js?1681084728064&_=1681084728064:2
    __WEBPACK_AMD_DEFINE_RESULT__</_callee/_callee$/</< https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    l https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:14
    fireWith https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:14
    G https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    onreadystatechange https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    send https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    ajax https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    t https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    getScript https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    __WEBPACK_AMD_DEFINE_RESULT__</_callee/_callee$/< https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _callee$ https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    u https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    B https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    v https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    asyncGeneratorStep https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    o https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _asyncToGenerator https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _asyncToGenerator https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    renderApp https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    loadStyles https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
[KTL_KnackApp.js:104:13](https://ctrnd.com/Lib/KTL/KTL_KnackApp.js)
Uncaught TypeError: Knack.router.scene_view is null
    assembleFilterURL https://ctrnd.com/Lib/KTL/Prod/KTL-0.10.17.js:2591
    userFilters https://ctrnd.com/Lib/KTL/Prod/KTL-0.10.17.js:2585
    setInterval handler*Ktl/this.userFilters< https://ctrnd.com/Lib/KTL/Prod/KTL-0.10.17.js:2581
    Ktl https://ctrnd.com/Lib/KTL/Prod/KTL-0.10.17.js:3850
    KnackApp https://ctrnd.com/Lib/KTL/KTL_KnackApp.js:7
    loadKtl https://ctrnd.com/Lib/KTL/KTL_Start.js:67
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    y https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    i https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    js https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    KnackInitAsync https://appcdn.cloud-database.co/642ee87912e0ca00294f3798/custom/main.js?1681084728064&_=1681084728064:2
    __WEBPACK_AMD_DEFINE_RESULT__</_callee/_callee$/</< https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    l https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:14
    fireWith https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:14
    G https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    onreadystatechange https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    send https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    ajax https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    t https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    getScript https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:21
    __WEBPACK_AMD_DEFINE_RESULT__</_callee/_callee$/< https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _callee$ https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    u https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    B https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    v https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    asyncGeneratorStep https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    o https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _asyncToGenerator https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    _asyncToGenerator https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    renderApp https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    loadStyles https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:1
    n https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    setTimeout handler*r https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
    exports https://cdn1.cloud-database.co/namespace/scripts/k_074772407a8f8d7b46272b030fdc689ec805eb88.js:587
```

**Comment by cortexrd** [comment]
Thank you sir for reporting this. 
I just fixed the bug in the version **0.10.19**.  Can you try again, and let me know if it works.
Normand

**Comment by cscott** [comment]
I figured out that the problem was I was using the "basic setup" instructions at https://github.com/cortexrd/Knack-Toolkit-Library#no-time-to-read-all-this-now---how-about-a-quick-tryout and then the "simple method" at https://github.com/cortexrd/Knack-Toolkit-Library#disabling-a-feature -- when I switched to the "advanced method" I managed to get it to load.  Hope that helps.  I'll checkout the fix in 0.10.19 as well.

**Comment by cscott** [comment]
Yes, 0.10.19 fixed the problem!

**Comment by cortexrd** [comment]
Great, thanks again for your input. 
One more contributor to this project - check!

Norm

**Comment by cortexrd** [comment]
> I figured out that the problem was I was using the "basic setup" instructions at https://github.com/cortexrd/Knack-Toolkit-Library#no-time-to-read-all-this-now---how-about-a-quick-tryout and then the "simple method" at https://github.com/cortexrd/Knack-Toolkit-Library#disabling-a-feature -- when I switched to the "advanced method" I managed to get it to load. Hope that helps. I'll checkout the fix in 0.10.19 as well.

No the setup type was not related, but still, it was a good idea to try what you did.  What happened is that I forgot to add the new inlineEditActive function in the KTL_KnackApp,js file.  My bad.
